src/arch/armv8/psci.c

```diff
@@ -162,7 +165,7 @@ static int32_t psci_affinity_info_handler(unsigned long target_affinity,
 
 static int32_t psci_features_handler(uint32_t feature_id)
 {
-    int32_t ret = PSCI_E_NOT_SUPPORTED;
+    int32_t ret;
```

Comment:
just a nit pick: I think it's fine to leave this initialized even if the value is overwritten

Reply:
Fixed in commit 9d3994a088888aef92cfe7e357356b385923936b